mediator/esmFldsExchange_ufs_mod.F90

```diff
@@ -743,17 +801,34 @@ subroutine esmFldsExchange_ufs(gcomp, phase, rc)
     ! to lnd - states and fluxes from atm
     if ( trim(coupling_mode) == 'ufs.nfrac.aoflux') then
        allocate(flds(21))
-       flds = (/'Sa_z      ', 'Sa_topo   ', 'Sa_tbot   ', 'Sa_pbot   ', &
-                'Sa_shum   ', 'Sa_u      ', 'Sa_v      ', 'Sa_pslv   ', &
-                'Faxa_lwdn ', 'Faxa_swdn ', 'Faxa_snowc', 'Faxa_snowl', &
-                'Faxa_rainc', 'Faxa_rainl', 'Faxa_rain ', 'Faxa_swnet'/)
+       if (lnd_name == 'lm4') then
```

Comment:
Hi Justin. Just a brief glance through this for now, but one thing that pops out is the overlap in the fields here. I think it would be better to collect those fields which do not overlap and those that do into separate code blocks.

Reply:
Hi Denise, I didn't think a draft PR would get review attention so quickly.  I am planing on cleaning this file up, including exactly as you say. Should I close for now until it's ready for review?

Reply:
No, leave it open; it's fine. We happened to have our bi-weekly component coordination meeting today, so I took a quick glance at it in order to report if it would be targeted for the next two weeks.

Reply:
> Hi Justin. Just a brief glance through this for now, but one thing that pops out is the overlap in the fields here. I think it would be better to collect those fields which do not overlap and those that do into separate code blocks.

This is now addressed with dfb1581e